cmd/goal/clerk.go

```diff
@@ -69,6 +69,7 @@ var (
 	requestOutFilename           string
 	simulateAllowEmptySignatures bool
 	simulateAllowMoreLogging     bool
+	simulateExtraBudget          uint64
```

Comment:
I think we should continue with "Allow". perhaps this is `AllowExtraOpcodes`. (while "Budget" is a useful term, I think "Opcode" is needed, to immediately convey that this is not about fees. `AllowExtraOpcodeBudget` would be fine if we want to be very explicit.

Reply:
I vote for AllowExtraOpcodeBudget

Reply:
see [extra-app-budget -> extra-opcode-budget, overrides ptr to instance ui…](https://github.com/algorand/go-algorand/pull/5354/commits/d305f968408260845418af128fa4d5de54ddc1e2)